### ISSUE TITLE
reduce loading time of user wallet / profile

### ISSUE THREAD

**Issue by kasumi-1** [semperai/effectiveacceleration.ai#34]
currently this causes rendering delay on first page load, as well as resizing

**Comment by AlternateIf** [comment]
@kasumi-1 upon creating my profile (settings username etc) i was able to sign the transaction. it went through without problems but the profile page was still open with it saying something like "waiting" or "loading" (don't know the exact term anymore)

after reloading after 30ish seconds everything was good. i guess this may be realted to the loading time and it not checking if the transaction has already confirmed

**Comment by jojedev** [comment]
@AlternateIf Hi! nice to meet you :)

Can you give me more info about this? when you say "upon creating my profile" I assume you're talking about /register page right? If you can give more info would appreciate it! :)

**Comment by AlternateIf** [comment]
@jojedev yes that's right. it's the modal where i set my username. i can try this with a different wallet tmr to give you more info about when the modal opens

**Comment by jojedev** [comment]
@AlternateIf @kasumi-1 I actually managed to sign up with now problems, please check this video and tell me what else should I do to replicate this issue. Thanks! :)
https://github.com/user-attachments/assets/64081fa2-b299-477c-b392-c8d2748b5dea

**Comment by AlternateIf** [comment]
hey @jojedev 

tried it with a fresh metamask wallet and everything worked just fine. i had the issue with a rainbow wallet. you can ignore this. if i manage to reproduce it i'll let you know :)
